README.md

```diff
@@ -6,6 +6,21 @@ The Proof of Concept Data Marketplace built using MAM and IOTA Tangle.
 
 Before you can run this application, you will need to install `Node.js` on your machine. Once you've installed `Node.js`, you can use `npm` to run commands listed below.
 
+### Create a Firebase Account
+
+The data-marketplace uses [Firebase](https://firebase.google.com/) as storage solution. 
+You need an active Firebase project to use the data-marketplace.
+
+### change config.json.example
+
+There is a json example in the src directory. Rename the config.json.example to config.json. 
+Replace all values with the values from your Firebase project.
+You can get get the information by the Firebase getting started guide in your project.
```

Comment:
```suggestion
You can get the information by using the Firebase [getting started guide](https://firebase.google.com/docs/) in your project.
```

Reply:
@rajivshah3 Good catch. 
It is not this getting started guide. It is the getting started guide in the Firebase console, under your project. Maybe I should add a screenshot.